### PR TITLE
Add unit test for MolGraph.get_formula after fixing bug in same 

### PR DESCRIPTION
**Pull request by mefuller**
(and one other little typo in a comment)

### CODE REVIEW

File: arc/species/speciesTest.py [hunk 1]
```diff
@@ -580,9 +580,9 @@ def test_process_xyz(self):
         """Test the process_xyz() function"""
         # the last four H's in xyz hve a leading TAB character which is removed when processing
         xyz = """
-
```
Comment (alongd) [inline arc/species/speciesTest.py:583]: these "tabs" were added intentionally (perhaps deleted here unintentionally?)
the idea is to make sure ARC is robust in reading in XYZ's even if those were pasted from an online source which often has "tab" chars. Same for the lines below.
Could you remove these changes from the commit? ("squash")

File: arc/species/speciesTest.py [hunk 2]
```diff
@@ -1375,6 +1375,19 @@ def test_from_dict(self):
         self.assertEqual(tsg.method, 'autotst')
         self.assertTrue(isinstance(tsg.rmg_reaction, Reaction))
 
+    def test_xyz_to_2d_get_formula(self):
+        from arc.species.xyz_to_2d import MolGraph
```
Comment (alongd) [inline arc/species/speciesTest.py:1379]: Could you relocate the import line to the top of the module? (we sort them there alphabetically)
This way additional unit tests we'll add will also have access to `MolGraph`

File: arc/species/xyz_to_2d.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 class Atom(object):
     """
-    Represents an atopm in a molecular graph.
+    Represents an atom in a molecular graph.
```
Comment (alongd) [inline arc/species/xyz_to_2d.py:20]: thanks!

File: arc/species/xyz_to_2d.py [hunk 4]
```diff
@@ -144,13 +144,14 @@ def get_formula(self):
         formula = ''
         if 'C' in list(elements.keys()):
             count = elements['C']
+            print(count)
```
Comment (alongd) [inline arc/species/xyz_to_2d.py:147]: a debug leftover?

File: arc/species/xyz_to_2d.py [hunk 5]
```diff
@@ -144,13 +144,14 @@ def get_formula(self):
...
             formula += 'C{:d}'.format(count) if count > 1 else 'C'
             del elements['C']
             if 'H' in list(elements.keys()):
                 count = elements['H']
                 formula += 'H{:d}'.format(count) if count > 1 else 'H'
                 del elements['H']
-        keys = elements.keys()
+        keys = list(elements.keys())
```
Comment (alongd) [inline arc/species/xyz_to_2d.py:154]: nice catch!

**Review by alongd — COMMENTED** [review]
Thanks for this contribution!
Please take a look at some minor comments I added

File: arc/species/speciesTest.py (hunk 1 above)
Comment (mefuller) [reply arc/species/speciesTest.py:583]: Is there an easy way to do line-by-line modifications here?

Comment (mefuller) [reply arc/species/speciesTest.py:583]: Looks like my linter ate those and never told me...

File: arc/species/xyz_to_2d.py (hunk 5 above)
Comment (mefuller) [reply arc/species/xyz_to_2d.py:153]: the function did not work without this fix

File: arc/species/speciesTest.py (hunk 1 above)
Comment (mefuller) [reply arc/species/speciesTest.py:583]: The easiest solution was to put back tabs (not sure why they show as different length - I thought I checked this) - I hope this is still ok

Comment (alongd) [reply arc/species/speciesTest.py:583]: You can perhaps reset soft (git reset --soft official/master) to remove the commits but keep all the code modifications (so the changes will only be staged but uncommited), then select just the particular lines you'd like to commit, e.g. using [git cola](https://git-cola.github.io/) gui.

File: arc/species/xyz_to_2d.py (hunk 5 above)
Comment (alongd) [reply arc/species/xyz_to_2d.py:153]: It's probably a fossil of the Python 2 -> 3 transition. Thanks for the fix!

**Review by alongd — APPROVED** [review]
Thanks for this contribution!

**Pull request merged**